web/massastation/src/pages/Index/Index.tsx

```diff
@@ -1,20 +1,126 @@
+import { ReactComponent as Image5Dark } from '../../assets/subduedImages/dark/5.svg';
+import { ReactComponent as Image1Light } from '../../assets/subduedImages/light/1.svg';
+import { ReactComponent as Image2Light } from '../../assets/subduedImages/light/2.svg';
+import { ReactComponent as Image3Light } from '../../assets/subduedImages/light/3.svg';
+import { ReactComponent as Image4Light } from '../../assets/subduedImages/light/4.svg';
+import { ReactComponent as Image5Light } from '../../assets/subduedImages/light/5.svg';
+import { FiCodepen, FiGlobe } from 'react-icons/fi';
+import { useNavigate } from 'react-router-dom';
+import { useEffect, useState } from 'react';
+import {
+  PluginHomePage,
+  PluginStoreItemRequest,
+} from '../../../../shared/interfaces/IPlugin';
+import { usePost, useResource } from '../../custom/api';
 
 export function Index() {
+  const navigate = useNavigate();
+  const [pluginWalletIsInstalled, setPluginWalletIsInstalled] = useState(false);
+  const [dashboardKey, setDashboardKey] = useState(0);
+  const [urlPlugin, setUrlPlugin] = useState('');
+
+  const { data: plugins } = useResource<PluginHomePage[]>('plugin-manager');
+
+  const { data: availablePlugins } =
+    useResource<PluginStoreItemRequest[]>('plugin-store');
+
+  const { mutate } = usePost<any, any>(`plugin-manager?source=${urlPlugin}`);
+
+  useEffect(() => {
+    const isWalletInstalled = plugins?.some(
+      (plugin: PluginHomePage) => plugin.name === 'wallet',
+    );
+    setPluginWalletIsInstalled(Boolean(isWalletInstalled));
+    if (!isWalletInstalled && availablePlugins) {
+      const walletPlugin = availablePlugins.find(
+        (plugin: PluginStoreItemRequest) => plugin.name === 'MassaWallet',
+      );
+      if (walletPlugin) {
+        setUrlPlugin(walletPlugin.file.url);
+      }
+    }
+  }, [plugins, availablePlugins]);
+
+  useEffect(() => {
+    setDashboardKey((prevKey) => prevKey + 1);
+  }, [pluginWalletIsInstalled]);
+
+  function handleInstallPlugin() {
+    try {
+      mutate({});
+      setPluginWalletIsInstalled(true);
+    } catch (error) {
+      console.error('Error installing plugin:', error);
+    }
+  }
+
   return (
-    <DashboardStation
-      imagesDark={[]}
-      imagesLight={[]}
-      components={[
-        <PluginWallet
-          isActive={true}
-          title={'Massawallet'}
-          iconActive={null}
-          iconInactive={null}
-          onClickActive={() => console.log('install')}
-          onClickInactive={() => console.log('launch')}
-        />,
-      ]}
-    />
+    <>
+      <div className="pt-32 pb-16 text-f-primary mas-banner">
+        Decentralization made easy
+      </div>
+      <div className="flex space-x-8 pb-10">
+        <Button
+          preIcon={<FiGlobe />}
+          customClass="w-96"
+          onClick={() => navigate('/web/search')}
+        >
+          <div className="flex items-center mas-buttons">
+            Search Massa ecosystem
```

Comment:
Can we use the I18n!?

You can define the keys here: `web/massastation/src/i18n/en_US.json`

then import:

`import Intl from '../../i18n/i18n';`

then use it:

`{Intl.t('pagename.function.key')}`

Let me know if you need help with that

Reply:
oh yes, I see what you mean! Nice!
It's done